### ISSUE TITLE
[buddy-server] Allow serving multiple directory paths

### ISSUE THREAD

**Issue by mariuslundgard** [popeindustries/buddy#113]
In addition to bundling and serving the bundled files, I’d like to serve files that are not built by `buddy`, e.g.:

```json
  ...
  "server": {
    "directory": [
      "www",
      "assets/fonts"
    ],
    "port": 8080
  }
  ...
```

Is there currently a way to do this with `buddy-server`?

**Comment by popeindustries** [comment]
Not currently, but should easy to add

**Comment by popeindustries** [comment]
Should work now in `buddy-server` 3.0.0 and `buddy` 6.13.0

**Comment by mariuslundgard** [comment]
Cool! Will check it out later tonight 👍

**Comment by mariuslundgard** [comment]
Thanks for helping me get sh*t done @popeindustries <3 It works like a charm.

The output looks a little different now? The `../../../..` part confuses me here:

```
...
done build in 136ms

live-reloading on port: 35729
serving ../../../.. at http://localhost:8080
```

**Comment by popeindustries** [comment]
Hmm, yeah, that doesn't look right. I'll make a note to fix that!

**Comment by popeindustries** [comment]
should be fixed in 6.13.1. Let me know if it's still borked...

**Comment by mariuslundgard** [comment]
Looks fine now 👍
